test/index.ts

```diff
@@ -21,3 +26,78 @@ describe('Access the server', () => {
     expect(response.body.data.hello).to.be.eq('Hello, world!');
   });
 });
+
+describe('Mutation createUser', () => {
+  afterEach(async () => {
+    const userRepository = User.getRepository();
+    await userRepository.clear();
+  });
+
+  it('Run mutation createUser', async () => {
```

Comment:
Pra ficar com mais cara de teste automatizado, nós convencionamos um texto que tem um formato específico iniciando com `should`. Por exemplo:

```ts
  it('should create user on database and return info ', async () => {
```

Reply:
feito